exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/conv/DummyConvertTo.java

```diff
@@ -32,6 +34,8 @@
 @FunctionTemplate(name = "convert_to", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
 public class DummyConvertTo implements DrillSimpleFunc {
 
+  @Param VarCharHolder in1;
```

Comment:
convert_to should take arbitrary type, and convert to binary type. Why does it only accept for VarChar?